SPDX-License-Identifier: BSD-3-Clause

OpenBSD's hidapi package renames hid_init to avoid a usbhid.h conflict.

Index: src/device/device_io_hid.cpp
--- src/device/device_io_hid.cpp.orig
+++ src/device/device_io_hid.cpp
@@ -92,7 +92,11 @@ namespace hw {
  
     void device_io_hid::init() {
       int r;
+#ifdef __OpenBSD__
+      r = hidapi_hid_init();
+#else
       r = hid_init();
+#endif
       ASSERT_X(r>=0, "Unable to init hidapi library. Error "+std::to_string(r)+": "+safe_hid_error(this->usb_device));
     }
 
